src/components/Dropdown/Dropdown.module.css

```diff
@@ -0,0 +1,104 @@
+.dropdown {
+  display: inline-block;
+  position: relative;
+  text-overflow: ellipsis;
+  user-select: none;
+  white-space: nowrap;
+}
+
+.dropdownButton,
+.dropdownButton:active,
+.dropdownButton:focus,
+.dropdownButton:hover {
+  align-items: center;
+  background: inherit;
+  border: none;
+  color: inherit;
+  cursor: pointer;
+  display: flex;
+  font-size: initial;
+  overflow-x: hidden;
+  padding: 0;
+}
+.dropdownButton:active,
+.dropdownButton:focus,
+.dropdownButton:hover {
+  color: #113;
+}
+
+/* caret */
+.dropdownButton::before {
+  content: "\25bc";
+  display: inline-block;
+  font-size: 10px;
+  margin-right: 4px;
+  transform: rotate(-90deg);
+  transition: transform 0.1s;
+}
+.open .dropdownButton::before {
+  transform: rotate(0deg);
+}
+
+/* alignment */
+.dropdownLeft .dropdownContent {
+  left: 0;
+}
+
+.dropdownContent {
+  background-color: #eee;
+  position: absolute;
+  right: 0;
+  border-radius: 6px;
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.2);
+  display: flex;
+  flex-direction: column;
+  max-height: 0;
+  max-width: 300px;
+  min-width: 160px;
+  transition: max-height 0.1s ease-out;
+  overflow-y: hidden;
+  z-index: 20;
+}
+
+.open .dropdownContent {
```

Comment:
nit: we can use `[aria-expanded="true"]` instead of `.open`

https://htmhell.dev/adventcalendar/2024/16/